docker-compose.yml

```diff
@@ -50,6 +50,12 @@ services:
       - ./:/go/src/app
       - ./.gocache:/go/pkg/mod
 
+  pact-publisher:
```

Comment:
Nit, but we might as well just call this `pact` since it technically does more than just publishing
```suggestion
  pact:
```